recipes-qt/qt5/qtwebengine_git.bb

```diff
@@ -26,6 +26,7 @@ DEPENDS += " \
     gperf-native \
     ${@bb.utils.contains('DISTRO_FEATURES', 'alsa', 'alsa-lib', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'libxcomposite libxcursor libxi libxrandr libxtst', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'pulseaudio', 'pulseaudio', '', d)} \
```

Comment:
Is there configure flag for pulseaudio support? It would be better with PACKAGECONFIG (which can be enabled based on DISTRO_FEATURES).

Reply:
Done